packages/example-apps/basic/src/MyComponent.ts

```diff
@@ -36,8 +38,7 @@ class MyComponent extends Component {
   @tracked count = 55;
 
   get currentLocale(): string {
-    return 'zh_CN';
-    // return (getHostMeta(this) as { locale: LocaleService }).locale.currentLocale;
+    return getOwner<{ services: { locale: LocaleService } }>(this).services.locale.currentLocale;
```

Comment:
For the example app, let's make an interface for the owner and then use it here.